docker/docker-compose.dev.yml

```diff
@@ -19,6 +19,7 @@ services:
       - db
     volumes:
       - ..:/owtf
+      - /owtf/owtf.egg-info
```

Comment:
lets remove this - egg info is only builds

Reply:
@viyatb Have made the changes.Thanks